doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -203,6 +203,10 @@ Bluetooth samples
     * Added energy consumption information to documentation.
     * Added a documentation section about distance offset calibration.
 
+* :ref:`peripheral_nfc_pairing` and :ref:`central_nfc_pairing` samples:
```

Comment:
Isn't the fix in the AC Parser library? Maybe it should rather be mentioned in the NFC section?

Reply:
Maybe we can mention it in both places?

Reply:
Both are OK for me.

Reply:
Ok I added notes in both places